packages/near-api-js/src/wallet-account.ts

```diff
@@ -102,7 +102,16 @@ export class WalletConnection {
             });
         }
         this._near = near;
-        const authDataKey = appKeyPrefix + LOCAL_STORAGE_KEY_SUFFIX;
+        const DEPRECATED_authDataKey = appKeyPrefix + LOCAL_STORAGE_KEY_SUFFIX;
```

Comment:
Should this case explicitly check for `'undefined' + LOCAL_STORAGE_KEY_SUFFIX`?  Otherwise if someone has passed an actual `appKeyPrefix`, we will be unnecessarily deleting the localstorage entry and re-creating it _every time this code runs_, right?

Reply:
Agreed 👍 , I think adding a `&& DEPRECATED_authDataKey !=  authDataKey` reads nicely here